From c499565879573224c3c6ba2be6bf2f097ceb496d Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Mon, 26 Feb 2018 17:48:49 +0200
Subject: [PATCH 0541/5242] MLK-10629-6 dts: imx7d-sdb: do not enable wlreg_on
 always

commit  2a1bdbf3205430c779832b100f0c61625e98d67a from
https://source.codeaurora.org/external/imx/linux-imx.git

Broadcom WiFi requires reset by operating wlreg_on regulator
when doing insmod/rmmod.
Keep wlreg_on will cause card state wrong and unable to re-insmod.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit d65683e1d0717fa039ebd4c55562e6ee745354b7)
Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index f86c072..53a89fe 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -144,8 +144,6 @@
 		regulator-name = "wlreg_on";
 		gpio = <&gpio4 21 GPIO_ACTIVE_HIGH>;
 		enable-active-high;
-		regulator-boot-on;
-		regulator-always-on;
 	};
 
 	panel {
-- 
1.7.9.5

